habitat_sim/utils/datasets_download.py

```diff
@@ -69,6 +69,18 @@ def initialize_test_data_sources(data_path):
             "link": data_path + "scene_datasets/coda",
             "version": "1.0",
         },
+        "replica_cad_dataset": {
+            "source": "https://dl.fbaipublicfiles.com/habitat/ReplicaCAD/ReplicaCAD_dataset_v1.0.zip",
+            "package_name": "ReplicaCAD_dataset_v1.0.zip",
+            "link": data_path + "replica_cad",
```

Comment:
Should it be `"link": data_path + "scene_datasets/replica_cad",` as we have it for mp3d/gibson?